guvnor-structure/guvnor-structure-backend/src/main/java/org/guvnor/structure/backend/organizationalunit/OrganizationalUnitServiceImpl.java

```diff
@@ -114,6 +114,11 @@ public OrganizationalUnit getOrganizationalUnit( final String name ) {
 
     @Override
     public Collection<OrganizationalUnit> getAllOrganizationalUnits() {
+        return new ArrayList<>( registeredOrganizationalUnits.values() );
+    }
+
+    @Override
+    public Collection<OrganizationalUnit> getOrganizationalUnits() {
```

Comment:
I tried not to break API compatibility, that's why I kept the same method name. Anyway, provided javadoc is quite self-explanatory.